commit c87bfe7b479591b14699b53057c7d84f0286411d
Author: djm <djm>
Date:   Tue Apr 12 05:39:35 2011 +0000

    s/recommended/required in warning:
    
    "It is recommended that your private key files are NOT accessible by others."
    
    since there is no way to skip this check; bz#1878

diff --git a/authfile.c b/authfile.c
index f2aec26..a49850c 100644
--- a/authfile.c
+++ b/authfile.c
@@ -606,7 +606,7 @@ key_perm_ok(int fd, const char *filename)
 		error("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@");
 		error("Permissions 0%3.3o for '%s' are too open.",
 		    (u_int)st.st_mode & 0777, filename);
-		error("It is recommended that your private key files are NOT accessible by others.");
+		error("It is required that your private key files are NOT accessible by others.");
 		error("This private key will be ignored.");
 		return 0;
 	}
